### ISSUE TITLE
notifyme no longer works

### ISSUE THREAD

**Issue by frennkie** [raspiblitz/raspiblitz#2231]
Just noticed that the sstmp package is no longer included in the updated base image.

```
admin@raspberrypi:~ $ sudo apt install ssmtp
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package ssmtp is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'ssmtp' has no installation candidate
```

Need to find an simple alternative.

**Comment by dennisreimann** [comment]
I also came across this while running `/home/admin/config.scripts/blitz.notify.sh on` after the v1.7.0 upgrade.

**Comment by vv01f** [comment]
`ssmtp` is deprecated, as a possible successor `msmtp` might be an option. any other suggestion? 
I also added sendxmpp for myself #2460

**Comment by frennkie** [comment]
Did you see #2232 ...?

**Comment by vv01f** [comment]
^^ no, I searched for msmtp and didnt find mstmp ;P. well, close my pr for now and wait for this to merge. then I can re-apply sendxmpp on that later.

**Comment by rootzoll** [comment]
PR merged to be included in 1.7.1RC2 ... set to final testing.

**Comment by rootzoll** [comment]
no bug reported on rc2 ... so closing for final release.
